lib/matplotlib/tests/test_axes.py

```diff
@@ -5757,6 +5759,17 @@ def test_empty_ticks_fixed_loc():
     ax.set_xticklabels([])
 
 
+@image_comparison(['set_label_coords.png'])
```

Comment:
Could this be `check_figures_equal` against a figure with `figtext` instead of the ylabel?

Reply:
I guess it could. Or check the actual location after using `set_label_coords`.